build-aux/docker/scan-go.sh

```diff
@@ -12,10 +12,10 @@ cd /app
 GO_VERSION=$(go version | sed -E 's/.*go([1-9\.]*).*/\1/')
 
 /scripts/go-mkopensource --output-format=txt --package=mod --output-type=markdown --gotar="$(ls /data/go*.src.tar.gz)" \
-  --unparsable-packages="$UNPARSABLE_PACKAGE_VALUE" >"${GO_DEPENDENCIES}"
```

Comment:
Where was set this env `UNPARSABLE_PACKAGE_VALUE`? I don't see where is initialized.

Reply:
I changed that variable to `UNPARSABLE_PACKAGE` and it's being set in the [dockerfile](https://github.com/datawire/go-mkopensource/blob/7b891b5471bc471241c2fcd617d552e5a5906d48/build-aux/docker/go_builder.dockerfile#L35)